schedule/security/create_hdd_common_criteria_for_mm.yaml

```diff
@@ -0,0 +1,9 @@
+name: create_hdd_common_criteria_for_mm
+description: >
+    This is used to publish the qcow2 file which is CC system role without
+    bridge network and all packages are needed have been installed.
```

Comment:
bridge network and all packages are needed have been installed. -> bridge network,  required packages for mm setup usage have been installed as well.

Reply:
Thanks, fixed